File: ClientApp/src/app/pagos/model/Pago.ts
export class Pago {
  pagoID: number;
  Concepto: string;
  Cant_Productos: number;
  Cliente: string;
  Empleado: string;
  Total: number;
  EstatusPago: number;
  constructor(
    pagoID?: number,
    Concepto?: string,
    Cant_Productos?: number,
    Cliente?: string,
    Empleado?: string,
    Total?: number,
    EstatusPago?: number
  ) {
    this.pagoID = pagoID!;
    this.Concepto = Concepto!;
    this.Cant_Productos = Cant_Productos!;
    this.Cliente = Cliente!;
    this.Empleado = Empleado!;
    this.Total = Total!;
    this.EstatusPago = EstatusPago!;
  }
}

File: Model/Pago.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace ExamenBancoBase.Model
{
    public class Pago
    {
        public int PagoID { get; set; }
        public string Concepto { get; set; }
        public int Cant_Productos { get; set; }
        public string Cliente { get; set; }
        public string Empleado { get; set; }
        public decimal Total { get; set; }
        [JsonConverter(typeof(StringEnumConverter))]
        public EstatusPagoEnum Estatus { get; set; }
    }
}
